crypto/fipsmodule/modes/xts.c

```diff
@@ -62,7 +62,6 @@ size_t CRYPTO_xts128_encrypt(const XTS128_CONTEXT *ctx,
                              uint8_t *out, size_t len, int enc) {
   union {
     uint64_t u[2];
-    uint32_t d[4];
     uint8_t c[16];
```

Comment:
Any idea why there isn't an issue mixing the uint64 and uint8? I get the issue with  `d` that you got rid of below by using `u` and a different shift.

Reply:
There is an issue with mixing them that is handled by loading the 64-bit values into separate variables, `tweak_u0` and `tweak_u1`, which was not needed in the little-endian case and the shifts were done directly on the union values.